_data/wai-evaluation-tools-list/submissions/c2512100-04d4-11f0-879c-311f258cbb73.json

```diff
@@ -0,0 +1,60 @@
+{
+  "a11yloc": "",
+  "actrules": "",
+  "assists": [
+    "Generating reports of evaluation results",
+    "Providing step-by-step evaluation guidance"
+  ],
+  "automated": [
+    "Component",
+    "Single page/screen",
+    "Sample of pages/screens",
+    "Entire website / software / product",
+    "Restricted or password protected pages"
+  ],
+  "comments": "",
+  "contact": "karthik@browserstack.com",
+  "desktop": [
+    "Android",
+    "iOS"
+  ],
+  "features": "Automate accessibility testing for mobile apps with just a single flag change. Run tests on real devices, catch violations early, and integrate seamlessly into your CI/CD pipeline. No maintenance needed—DOM updates are tracked automatically. Get detailed reports, monitor trends, and ensure your app stays accessible with every release.",
```

Comment:
"ensure your app stays accessible with every release" seems to suggest that automatic testing is sufficient to ensure that an app is accessible. This may be misleading.